libaleth-interpreter/VM.h

```diff
@@ -119,14 +118,14 @@ class VM
 
     const evmc_tx_context& getTxContext();
 
-    void throwOutOfGas();
-    void throwInvalidInstruction();
-    void throwBadInstruction();
-    void throwBadJumpDestination();
-    void throwBadStack(int _removed, int _added);
-    void throwRevertInstruction(owning_bytes_ref&& _output);
-    void throwDisallowedStateChange();
-    void throwBufferOverrun(intx::uint512 const& _enfOfAccess);
+    static void throwOutOfGas();
+    static void throwInvalidInstruction();
+    static void throwBadInstruction();
+    static void throwBadJumpDestination();
+    void throwBadStack(int _removed);
+    void throwRevertInstruction(uint64_t _offset, uint64_t _size);
```

Comment:
I'd fill `m_output` outside of the function and make it static as others

Reply:
This is accessed also in EVMC's execute() method. More refactoring is possible to clean up how EVMC result is created and reported.